crates/scarb-api/src/lib.rs

```diff
@@ -280,6 +299,48 @@ mod tests {
         );
     }
 
+    #[test]
+    fn is_required_package_version_test() {
+        let temp = setup_package("basic_package");
+
+        let manifest_path = temp.child("Scarb.toml");
+        manifest_path
+            .write_str(&formatdoc!(
+                r#"
+                [package]
+                name = "version_checker"
+                version = "0.1.0"
+
+                [[target.starknet-contract]]
+                sierra = true
+                casm = true
```

Comment:
[nit] - don't need that since we use our sierra-casm compiler

Reply:
Done